117.md

```diff
@@ -0,0 +1,121 @@
+
+The following tags MAY be present:
+
+- A `p` tag to notify the creator of the bounty that the applicant has applied.
+
+- A `timeline` tag to provide a date and time when the applicant expects to complete the bounty by.
+
+## Bounty Lifecycle
+
+The following is a description of the lifecycle of a bounty:
+
+1: **Open:** The bounty is created and published by the creator and accepting applications. The bounty MUST be created with an `s` tag of `open`.
+
+2: **Assigned:** The bounty has been assigned. The bounty MUST be updated with an `s` tag of `assigned` and a `p` tag of the applicant public key.
+
+3: **Completed:** The bounty has been completed by the applicant and the creator has accepted the work. This can be inferred by the client by looking up a zap receipt referencing the **Bounty Event** and the assigned user's public key. The **Bounty Event** content MUST match the content found in the description of the **Bounty Application Event**. The reward MUST match in the zap receipt, **Bounty Event** reward tag and the reward tag found in the description of the **Bounty Application Event**.
+
+4: **Canceled:** The bounty has been canceled by the creator. The bounty MUST be updated with an `s` tag of `canceled`.
+
+## Solution Submission
+
+Solutions to bounties may be submitted via a nip-04 encrypted message to the creator of the bounty. Or however the client chooses.
+
+## Motivation
+
+Bounties will provide social incentive to complete tasks related to the development of Nostr and allow developers to get paid and build a reputation across the network.
+
+## Why is the Bounty Event Replaceable?
+
+So that we can track the status of the bounty which provides a way for the client to filter out bounties that have already been assigned.
+
+We can work around this by making sure that the content and reward of the bounty event is the same as the content and reward found in the stringified **Bounty Event** in the **Bounty Event Application**. No **Bounty Event** without a matching reward across these three fields (including the zap receipt) will be considered complete. (Unless both parties agree to settle outside of the network)
+
+If the creator changes the content or reward in an effort to scam the applicant, the applicant has a record of the original content and reward in the **Bounty Event Application**. And the bounty will not count as complete which will at the very least not improve the creators reputation.
+
+If the applicant puts a fraudulent **Bounty Event** in the description of the **Bounty Event Application** the creator can choose not to pay.
+
+Without an agreed upon 3rd party there will always be a way to scam the system. This is the best way I can think of to discourage scammers.
+If the bounty was not replaceable, the creator could just delete the event anyway and skip paying after the applicant has submitted their work.
```

Comment:
That should also be possible to delete a replaceable.

Reply:
I guess what I mean to say here is that even if the bounty wasn't replaceable the creator could just delete their bounty.

Reply:
Ah now I understand, thanks.